tests/scripts/depends-hashes.pl

```diff
@@ -67,27 +71,29 @@ sub abort {
     exit 1;
 }
 
-for my $hash (@hashes) {
+for my $hash_config (@hash_configs) {
     system( "cp $config_h.bak $config_h" ) and die "$config_h not restored\n";
     system( "make clean" ) and die;
 
     print "\n******************************************\n";
-    print "* Testing hash option: $hash\n";
+    print "* Testing hash options: ", join(', ', @$hash_config), "\n";
     print "******************************************\n";
-    $ENV{MBEDTLS_TEST_CONFIGURATION} = "-$hash";
 
-    system( "scripts/config.py $hash" )
-        and abort "Failed to $hash\n";
+    for my $hash (@$hash_config) {
+        $ENV{MBEDTLS_TEST_CONFIGURATION} = "-$hash";
```

Comment:
This is setting an environment variable. It shouldn't be in the inner loop. The value of `MBEDTLS_TEST_CONFIGURATION` is only used in test logs, so its exact content doesn't matter. I think `$hash_config[0]` would be a good choice.